be/src/olap/tablet_meta.cpp

```diff
@@ -553,33 +523,6 @@ RowsetMetaSharedPtr TabletMeta::acquire_rs_meta_by_version(const Version& versio
     return rs_meta;
 }
 
-OLAPStatus TabletMeta::delete_inc_rs_meta_by_version(const Version& version) {
-    auto it = _inc_rs_metas.begin();
-    while (it != _inc_rs_metas.end()) {
-        if ((*it)->version().first == version.first
-              && (*it)->version().second == version.second) {
-            _inc_rs_metas.erase(it);
-            break;
-        } else {
-            it++;
-        }
-    }
-
-    return OLAP_SUCCESS;
-}
-
-RowsetMetaSharedPtr TabletMeta::acquire_inc_rs_meta_by_version(const Version& version) const {
-    RowsetMetaSharedPtr rs_meta = nullptr;
```

Comment:
It is used in incremental clone, if it is removed the clone logic may run to error.